src/xcode/ENA/ENAUITests/TicketValidation/ENAUITests_19_TicketValidation.swift

```diff
@@ -99,5 +107,52 @@ class ENAUITests_19_TicketValidation: CWATestCase {
 		snapshot("screenshot_ticket_validation_second_consent_3")
 		app.swipeUp()
 		snapshot("screenshot_ticket_validation_second_consent_4")
+		
+		/// navigate to the validation result screen
+		app.buttons[AccessibilityIdentifiers.General.primaryFooterButton].waitAndTap()
+		
+		/// check for the content on validation result screen
+		XCTAssertTrue(app.images[AccessibilityIdentifiers.TicketValidation.ValidationResult.Passed.headerImageWithTitle].waitForExistence(timeout: .short))
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.TicketValidation.ValidationResult.Passed.subtitle].waitForExistence(timeout: .short))
+		
+		/// take screenshot on validation result screen
+		snapshot("screenshot_ticket_validation_result_passed")
+	}
+	
+	func test_screenshot_ticketValidation_failed() throws {
+		app.setLaunchArgument(LaunchArguments.healthCertificate.secondHealthCertificate, to: true)
+		app.setLaunchArgument(LaunchArguments.ticketValidation.result.isFailed, to: true)
+		app.launch()
+		
+		app.buttons[AccessibilityIdentifiers.TabBar.scanner].waitAndTap()
+		
+		/// Simulator only Alert will open where you can choose what the QRScanner should scan
+		let ticketValidationButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.UniversalQRScanner.fakeTicketValidation])
+		ticketValidationButton.waitAndTap()
+		
+		/// check for the content on first consent screen
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.TicketValidation.FirstConsent.legalBox].waitForExistence(timeout: .short))
+		
+		/// navigate to certificate selection screen
+		app.buttons[AccessibilityIdentifiers.General.primaryFooterButton].waitAndTap()
```

Comment:
Please not that this could be a source of flakiness, if there are multiple layers and more than one "AccessibilityIdentifiers.General.primaryFooterButton" in the view hierarchy.